gluster_exporter/metric_brick.go

```diff
@@ -71,6 +76,15 @@ var (
 		},
 		brick_labels,
 	)
+
+	glusterSubvolEffectiveIUsed = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "gluster",
+			Name:      "subvol_effective_used",
```

Comment:
can be named similar to brick. `subvol_capacity_used`

Reply:
Ack